basic/okta_access_strategy/okta_access_strategy.tf

```diff
@@ -1,57 +1,13 @@
-provider "sym" {
-  org = "sym-example"
-}
-
-provider "aws" {
-  region = "us-east-1"
-}
-
-############ General AWS Secrets Manager Setup ##############
-
-# Creates an AWS IAM Role that the Sym Runtime can use for execution
-# Allow the runtime to assume roles in the /sym/ path in your AWS Account
-module "runtime_connector" {
-  source  = "symopsio/runtime-connector/aws"
-  version = ">= 1.0.0"
-
-  # The aws/secretsmgr addon is required to access secrets
-  addons = ["aws/secretsmgr"]
-
-  environment = "main"
-}
-
-# An Integration that tells the Sym Runtime resource which AWS Role to assume
-# (The AWS Role created by the runtime_connector module)
-resource "sym_integration" "runtime_context" {
-  type = "permission_context"
-  name = "main-runtime"
-
-  external_id = module.runtime_connector.settings.account_id
-  settings    = module.runtime_connector.settings
-}
-
-# This resource tells Sym which role to use to access your AWS Secrets Manager
-resource "sym_secrets" "this" {
-  type = "aws_secrets_manager"
-  name = "main-sym-secrets"
-
-  settings = {
-    context_id = sym_integration.runtime_context.id
-  }
-}
-
-############ Okta Integration and Secret Setup ##############
-
 # An AWS Secrets Manager Secret to hold your Okta API Key. Set the value with:
 # aws secretsmanager put-secret-value --secret-id "main/okta-api-key" --secret-string "YOUR-OKTA-API-KEY"
 resource "aws_secretsmanager_secret" "okta_api_key" {
-  name        = "main/okta-api-key"
+  name        = "sym/flow-name/okta-api-key"
   description = "API Key for Sym to call Okta APIs"
 
   tags = {
-    # This SymEnv tag is required and MUST match the `environment` in your `runtime_connector` module
-    # because the aws/secretsmgr only grants access to secrets tagged with a matching SymEnv value
-    SymEnv = "main"
+    # This SymEnv tag is required and MUST match the SymEnv tag in the 
+    # aws_iam_policy.secrets_manager_access in your `secrets.tf` file
+    SymEnv = local.environment_name
```

Comment:
The other examples use `"main"`, should we change all of those to use `local.environment_name`?